.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+orbs:
+  slack: circleci/slack@3.4.1
+version: 2.1
+jobs:
+  test:
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
can we use 18.04?

Reply:
I suppose not, it's not in the list: https://circleci.com/docs/2.0/configuration-reference/#available-machine-images

Reply:
damn, i thought only travis was using such obsolete thing